static/docs/commands-reference/pull.md

```diff
@@ -1,53 +1,116 @@
 # pull
 
-Pulls data files to the working space.
+Pulls data files to the local workspace from a remote DVC cache based on relative changes between the two.
```

Comment:
How about something like this?

`Downloads missing files and directories from a [remote storage]('doc/commands-reference/remote') to the local cache based on DVC files in the working space. Then links them into the working space.`